docs/docs/howto/encrypted-storage.md

```diff
@@ -9,177 +9,152 @@ This step is recommended for any Contrast deployment that stores sensitive data
-```
-
-### Deploy the Contrast runtime
-
-Contrast depends on a [custom Kubernetes `RuntimeClass`](../architecture/components/runtime.md),
-which needs to be installed to the cluster initially.
-This consists of a `RuntimeClass` resource and a `DaemonSet` that performs installation on worker nodes.
-This step is only required once for each version of the runtime.
-It can be shared between Contrast deployments.
-
-<Tabs queryString="platform">
-<TabItem value="metal-qemu-snp" label="Bare metal (SEV-SNP)">
-```sh
-kubectl apply -f https://github.com/edgelesssys/contrast/releases/latest/download/runtime-metal-qemu-snp.yml
-```
-</TabItem>
-<TabItem value="metal-qemu-tdx" label="Bare metal (TDX)">
-```sh
-kubectl apply -f https://github.com/edgelesssys/contrast/releases/latest/download/runtime-metal-qemu-tdx.yml
-```
-</TabItem>
-</Tabs>
-
-### Download the Contrast Coordinator resource
-
-Download the Kubernetes resource of the Contrast Coordinator, comprising a single replica deployment and a
-LoadBalancer service. Put it next to your resources:
-
-```sh
-curl -fLO https://github.com/edgelesssys/contrast/releases/latest/download/coordinator.yml --output-dir resources
-```
-
-### Generate policy annotations and manifest
-
-Run the `generate` command to generate the execution policies and add them as
-annotations to your deployment files. A `manifest.json` file with the reference values
-of your deployment will be created:
-
-<Tabs queryString="platform">
-<TabItem value="metal-qemu-snp" label="Bare metal (SEV-SNP)">
-
-```sh
-contrast generate --reference-values metal-qemu-snp resources/
-```
-
-On bare-metal SEV-SNP, `contrast generate` is unable to fill in the `MinimumTCB` values as they can vary between platforms and CPU models.
-They will have to be filled in manually.
-
-If you don't know the values from the firmware you installed, you can use the [`snphost`](https://github.com/virtee/snphost) tool to retrieve the current TCB.
-
-```sh
-snphost show tcb
-```
-```console
-Reported TCB: TCB Version:
-  Microcode:   72
-  SNP:         23
-  TEE:         0
-  Boot Loader: 9
-  FMC:         None
-Platform TCB: TCB Version:
-  Microcode:   72
-  SNP:         23
-  TEE:         0
-  Boot Loader: 9
-  FMC:         None
-```
-
-Use the values from `Platform TCB` to fill in the `MinimumTCB` values in the generated `manifest.json` file.
+The following demonstrates how to set up an encrypted LUKS mount for an arbitrary `/mount/path` directory to easily deploy an application with encrypted persistent storage using Contrast.
+
+### Configuration
+
+Add a `volume` to your pod or stateful set (called `mount-name` in the example configuration below)
+which references your encrypted, persistent storage device either through one of the [supported volume types](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#raw-block-volume-support).
```

Comment:
I just realized that the purpose of this volume is not clear enough and we're mixing the emptyDir and the block device. We should rearrange the docs like this:

> Add a volume of type emptyDir to your pod. Mount it with HostToCOntainer propagation. This is where the initializer will mount the encrypted volume.
>
> Add a block device volume to the pod. The initializer creates an encrypted volume and a filesystem on this device.